9_2Channel_LCDBarGraph/2Channel_LCDBarGraph.ino

```diff
@@ -0,0 +1,247 @@
+// 2Channel_LCDBarGraph- BioAmp EXG Pill
+// https://github.com/upsidedownlabs/BioAmp-EXG-Pill
+// https://github.com/upsidedownlabs/Muscle-BioAmp-Arduino-Firmware
+
+// Upside Down Labs invests time and resources providing this open source code,
+// please support Upside Down Labs and open-source hardware by purchasing
+// products from Upside Down Labs!
+
+// Copyright (c) 2021 - 2024 Upside Down Labs - contact@upsidedownlabs.tech
+// Copyright (c) 2021 - 2024 Aryan Prakhar - aryanprakhar1010@gmail.com
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <LiquidCrystal_I2C.h>

```

Comment:
Provide the library author name and link if users cannot decide which library to install.

Reply:
I have mentioned the library author name as well as their github source link. Please review